### ISSUE TITLE
Theme with Accessibility

### ISSUE THREAD

**Issue by foxmask** [havefnubb/havefnubb#34]
The default theme is not WCAG compliant.

Have to deal with this !

Changes in progress : 
-  missing keywords and language in meta header
-  colors will be more mixed (contrastée)
-  Font size will be more readable (less small)
